EXT_NRAYTRACING.md

```diff
@@ -0,0 +1,114 @@
+Domain-Specific Naming Conventions for Neutron raytracing codes
+===============================================================
+
+openPMD extension name: `NRAYTRACING`
+
+
+Introduction
+------------
+
+This extention is designed for transfer of neutron particle states between neutron raytracing simulation codes.
+The neutrons are saved in rays with a given weight corresponding to expected intensity.
+This standard is not yet finished and is currently based on EXT_BeamPhysics / EXT_WAVEFRONT
+
+Each record needs to have the same length, as the n'th index for each record correspond to particle n.

```

Comment:
remove since not accurate anymore

Reply:
Why is this not relevant anymore? Since for example position is stored as three separate lists, they have to be the same length to make sense.